### PR TITLE
Add oidc_e2e_setup_py for each supported Python version (#43)

### LINKED ISSUE #43: need new oidc_e2e* image to cover django 4
Django 4 supports Python >= 3.8, but the `oidc_e2e_setup_py3` image we're creating now uses Python 3.7. Traditionally, we try to have these images at the minimum-supported version, but that won't work with Django 4.

### PR DESCRIPTION
**Pull request by willkg**
Previously, we had an oidc_e2e_setup_py2 and an oidc_e2e_setup_py3 which
sufficed for our testing needs. After we dropped support for Python 2.7,
we dropped the oidc_e2e_setup_py2.

However, Django 4 requires Python >= 3.8 and we want to have a Python
3.7 image.

This changes our support a bit so now we're going to have an image for
every supported Python version. This adds the missing images and updates
the documentation accordingly.

This also updates the Python images used to be pinned to a specific
Python version (e.g. 3.7.13) and also update them to use bullseye
(current Debian stable) since stretch is not longer supported.

Fixes #43

### CODE REVIEW

**Comment by willkg** [comment]
@akatsoulas I'm pretty sure this will work for you. Does it look ok?

File: dockerfiles/oidc_e2e_setup_py310 [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM python:3.7-stretch
+FROM python:3.10.4-bullseye
```
Comment (willkg) [inline dockerfiles/oidc_e2e_setup_py310:1]: stretch is no longer supported, so this updates the images to bullseye which is the current debian stable.

**Review by akatsoulas — APPROVED** [review]
Looks good!

**Comment by willkg** [comment]
Thank you! I'll do a release now.

**Pull request merged**